appveyor.yml

```diff
@@ -15,7 +15,7 @@ environment:
     # See: http://stackoverflow.com/a/13751649/163740
     CMD_IN_ENV: 'cmd /E:ON /V:ON /C .\tests\scripts\appveyor\run_with_env.cmd'
 
-  PYTEST: py.test --maxfail 5 --durations=10 --junitxml=junit-results.xml
+  PYTEST: py.test -n3 --maxfail 5 --durations=10 --junitxml=junit-results.xml
```

Comment:
Just to be sure we're on the same page, the root case I saw (some time ago) when playing with the `-n3` option was that the [xdist](https://github.com/pytest-dev/pytest-xdist) plugin didn't work reliably on Windows; it seemed to work quite well on Linux. I would, of course, be overjoyed if this works now, since Appveyor testing time is sooooo long.